portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,34 +26,47 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    private Map<String, String> comments = new HashMap<String, String>();
-
-    @Override
-    public void init() {
-        comments.put("Mario", "Hi!");
-        comments.put("Mario's imaginary friend", "Hi! What are you going?");
-        comments.put("Mario", "Just learning how to use JSON");
-        comments.entrySet().forEach(entry->{
-            System.out.println(entry.getKey() + ": " + entry.getValue());  
-        });
-    } 
-
-    @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String json = convertToJson(comments);
-        System.out.println("Sending string: " + json);
-        response.setContentType("application/json;");
-        response.getWriter().println(json);
-    }
+  // IMPROVE: redundently storing display names
+  private List<List<String>> comments = new ArrayList<List<String>>();
```

Comment:
Another way to do this would be to add a new class
Class Row {
  String key;
  String value;
}
and then have a List<Row>.
But since the next step is to write everything to datastore, maybe leaving it this way should be fine.

Reply:
Thank you